mkplatformfs.sh.in

```diff
@@ -118,9 +118,10 @@ case "$PLATFORM" in
     dockstar*) PKGS="$BASEPKG ${PLATFORM%-*}-base" ;;
     odroid-u2*) PKGS="$BASEPKG ${PLATFORM%-*}-base" ;;
     odroid-c2*) PKGS="$BASEPKG ${PLATFORM%-musl}-base" ;;
-    rpi3*) PKGS="$BASEPKG rpi-base" ;;
-    rpi2*) PKGS="$BASEPKG rpi-base" ;;
-    rpi*) PKGS="$BASEPKG ${PLATFORM%-*}-base" ;;
+    rpi1*) PKGS="$BASEPKG rpi-base rpi1-kernel" ;;
```

Comment:
Renamed platform from `rpi` to `rpi1`, which may or may not break other things; it caused `./mkplatformfs.sh rpi ...` to fail with `FATAL: mkplatformfs.sh: invalid platform!`

Reply:
indeed

Reply:
yeah this breaks other things.  In general you should prefer to leave as it was and call later generations with a number.

Reply:
will do, thanks for review